src/app/cube/new-home/learning-object-info/hierarchies/hierarchies.component.html

```diff
@@ -1 +1,27 @@
-<p>hierarchies works!</p>
+<div class="hierarchy-container">
+  <div class="vertical-container">
+    <h1>Leverages the Bloom's taxonomy for measureable results</h1>
+    <!-- Dummy text until the full text is provided for this section -->
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+    <br>
+    <h1>Related curricula grouped hierarchically</h1>
+    <!-- Dummy text until the full text is provided for this section -->
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
```

Comment:
We need to replace the lorem ipsums with real descriptions.  I'd ask Paige or Mike about this.